pkg/services/live/pushws/ws.go

```diff
@@ -87,21 +87,24 @@ func setupWSConn(ctx context.Context, conn *websocket.Conn, config Config) {
 	}
 	if pingInterval > 0 {
 		pongWait := pingInterval * 10 / 9
-		_ = conn.SetReadDeadline(time.Now().Add(pongWait))
-		conn.SetPongHandler(func(string) error {
+		// _ = conn.SetReadDeadline(time.Now().Add(pongWait))
```

Comment:
^^ commenting this out lets it work, but that seems wrong

This code is copied from:
https://github.com/centrifugal/centrifuge/blob/master/_examples/unidirectional_ws/main.go#L481

Reply:
@FZambia any ideas what may be happening?  or where to look?

Reply:
Let me check, probably tomorrow. I think these two mentioned issues in the description are different though?

Reply:
thank you thank you!

I can reproduce https://github.com/grafana/grafana/issues/60905 with the simple standalone websocket sender -- but have not dug into https://github.com/grafana/grafana/issues/67394 -- it seems like a frontend equivolent.

Reply:
Regarding #60905 answered [in the issue](https://github.com/grafana/grafana/issues/60905#issuecomment-1527858730).